plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -18,14 +18,41 @@ limitations under the License.
 package defaults
 
 import (
+	"os"
+	"strconv"
+
 	"k8s.io/kubernetes/pkg/util/sets"
 	"k8s.io/kubernetes/plugin/pkg/scheduler"
 	"k8s.io/kubernetes/plugin/pkg/scheduler/algorithm"
 	"k8s.io/kubernetes/plugin/pkg/scheduler/algorithm/predicates"
 	"k8s.io/kubernetes/plugin/pkg/scheduler/algorithm/priorities"
 	"k8s.io/kubernetes/plugin/pkg/scheduler/factory"
+
+	"github.com/golang/glog"
 )
 
+// Amazon reccomends having no more that 40 volumes attached to an instance,
+// and at least one of those is for the system root volume.
+const DefaultMaxEBSVolumes = 39
+
+// GCE instances can have up to 16 PD volumes attached.
+const DefaultMaxGCEPDVolumes = 16
+
+// getMaxVols checks the max PD volumes environment variable, otherwise returning a default value
+func getMaxVols(defaultVal uint) uint {
```

Comment:
AIUI Go style is to avoid using "unit" except in very particular situations like bitmasks. I would suggest changing all of this to just regular int.